### ISSUE TITLE
plugin does not recognize OBS Studio 28/29 is installed

### ISSUE THREAD

**Issue by Nrde** [Loupedeck-open-source/Loupedeck-ObsStudio-OpenPlugin#13]
The plugin always says OBS is not running/installed.

When looking at the source something like GetWindowsInstallationRoot does not find my OBS installation. I don't have path "OBS Studio" in my windows registry.

**Comment by laperie** [comment]
How did you install OBS - was it using an official OBS installer?

**Comment by Nrde** [comment]
Yes, although I can't say how I originally installed it, since I've been
upgrading it since version 22 or so.

I can try to uninstall and installing it from scratch if you think it would
help.

ke 11. tammik. 2023 klo 13.42 Andrei Laperie ***@***.***)
kirjoitti:

**Comment by laperie** [comment]
yes, if possible please try reinstalling OBS. Note, however, that you'll also need to reinstall Loupedeck so that the Loupedeck OBS connector ends up in the OBS plugin folder. After starting OBS you should see 'Loupedeck connector' in OBS Tools menu

**Comment by Nrde** [comment]
OBS re-install fixed the problem. And now I have OBS studio reg key in the
registry. I had last installed OBS fully in version 24 it seems, and only
updates after.

Maybe you can add a note about possible problem with that.

ke 11. tammik. 2023 klo 14.04 Andrei Laperie ***@***.***)
kirjoitti:

**Comment by laperie** [comment]
> OBS re-install fixed the problem. And now I have OBS studio reg key in the registry. I had last installed OBS fully in version 24 it seems, and only updates after. Maybe you can add a note about possible problem with that. ke 11. tammik. 2023 klo 14.04 

Done in Readme (we also have https://support.loupedeck.com/cannot-connect-to-obs )
